src/org/zaproxy/zap/extension/fieldenumeration/FieldEnumeration.java

```diff
@@ -0,0 +1,133 @@
+package org.zaproxy.zap.extension.fieldenumeration;
+
+import java.io.*;
+import java.util.*;
+import java.lang.*;
+
+import java.awt.Frame;
+import java.awt.HeadlessException;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+
+import javax.swing.JLabel;
+import javax.swing.JPanel;
+import javax.swing.JRadioButton;
+import javax.swing.AbstractButton;
+import javax.swing.ButtonGroup;
+import javax.swing.JButton;
+import javax.swing.JOptionPane;
+
+import org.apache.log4j.Logger;
+import org.parosproxy.paros.db.DatabaseException;
+import org.parosproxy.paros.Constant;
+import org.parosproxy.paros.extension.AbstractDialog;
+import org.parosproxy.paros.model.HistoryReference;
+import org.parosproxy.paros.model.Model;
+import org.parosproxy.paros.network.HtmlParameter;
+import org.parosproxy.paros.network.HttpMalformedHeaderException;
+import org.zaproxy.zap.view.messagecontainer.http.HttpMessageContainer;
+
+public class FieldEnumeration extends AbstractDialog {
+
+	private static final long serialVersionUID = 1L;
+	private static final Logger logger = Logger.getLogger(FieldEnumeration.class);	
```

Comment:
This class is totally new (there are places [unfortunately] in the code base this wasn't done from the start), this variable is a constant (static final) the name should be full caps: `LOGGER`